Tests/ttLib/tables/_g_l_y_f_test.py

```diff
@@ -158,3 +162,63 @@ def test__checkFloat_overflow(self):
         g.append((0x8000, 0))
         assert g.array.typecode == "d"
         assert g.array == array.array("d", [1.0, 1.0, 32768.0, 0.0])
+
+
+CURR_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+DATA_DIR = os.path.join(CURR_DIR, 'data')
+
+GLYF_TTX = os.path.join(DATA_DIR, "_g_l_y_f_outline_flag_bit6.ttx")
+GLYF_BIN = os.path.join(DATA_DIR, "_g_l_y_f_outline_flag_bit6.glyf.bin")
+HEAD_BIN = os.path.join(DATA_DIR, "_g_l_y_f_outline_flag_bit6.head.bin")
+LOCA_BIN = os.path.join(DATA_DIR, "_g_l_y_f_outline_flag_bit6.loca.bin")
+MAXP_BIN = os.path.join(DATA_DIR, "_g_l_y_f_outline_flag_bit6.maxp.bin")
+
+
+def strip_ttLibVersion(string):
+    return re.sub(' ttLibVersion=".*"', '', string)
+
+
+class glyfTableTest(unittest.TestCase):
```

Comment:
the rest of the tests in this file already use pytest-style tests (with plain `assert` statements), I would prefer that over unittest-style tests.. but anyway some tests is better than none, thanks anyway.